library/Freckle/App/Database.hs

```diff
@@ -130,6 +130,7 @@ data PostgresConnectionConf = PostgresConnectionConf
   , pccDatabase :: String
   , pccPoolSize :: Int
   , pccStatementTimeout :: PostgresStatementTimeout
+  , pccSchema :: String
```

Comment:
I don't think we should require it. I think we should ether make this `Maybe String`, or set a `Env.def "public"` in the parser.